music_metadata_extractor/sources/youtube/scraper.py

```diff
@@ -46,4 +50,6 @@ def scrape_embedded_yt_metadata(soup) -> BaseProviderInput:
     return DictInput(
         song_name=check_key_get_value(info, "Song"),
         artist_name=check_key_get_value(info, "Artist"),
+        song_ytviews= int(soup.find("div", class_="watch-view-count").text[:-6].replace(",","")),
+        song_ytdate = dt.strptime(soup.find("strong", class_="watch-time-text").text[-11:],"%d-%b-%Y")
```

Comment:
[DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) : ) Add a private function that will parse these details and use it in both places you're doing this operation.

Reply:
Ok. I'm being lazy ig. Blame the quarantine.